teradatalabs/centos6-ssh-oj8/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM jdeathe/centos-ssh:centos-6-1.2.0
+FROM jdeathe/centos-ssh:centos-6
```

Comment:
Thoughts about pinning this to the most recent (centos-6-1.8 or something)? We may get out of date again, but at least we won't have arbitrary failures if jdeathe/centos-ssh changes.